services/src/main/scala/cromwell/pipeline/service/AuthService.scala

```diff
@@ -91,10 +91,11 @@ object AuthService {
         val refreshTokenContent = RefreshTokenContent(user.userId, None)
         getAuthResponse(accessTokenContent, refreshTokenContent, Instant.now.getEpochSecond)
       }
-
+      // scalastyle:off if.brace
```

Comment:
Sorry, I just noticed it. You need to remove it as well

Reply:
Missed that, sry. Fixed